templates/default/flyingduck.service.erb

```diff
@@ -5,10 +5,10 @@ Requires = docker.service <%= @local_dependencies %>
 
 [Service]
 Environment=FLYINGDUCK_MEMORY=<%= node['flyingduck']['memory_gbs'] %>
-Environment=FLYINGDUCK_CPUS=<%= node['flyingduck']['cpus'] %>
+Environment=FLYINGDUCK_CPU_LIMIT=<%= node['flyingduck']['cpus'] %>
 ExecStart = /usr/bin/docker run --rm --name %n \
       --memory=${FLYINGDUCK_MEMORY}G \
-      --cpus=${FLYINGDUCK_CPUS} \
+      --cpus=${FLYINGDUCK_CPU_LIMIT} \
```

Comment:
We should remove this and add it to the line above as follows 
`--memory=${FLYINGDUCK_MEMORY}G ${FLYINGDUCK_CPU_LIMIT}`